app/controllers/cat_controller.rb

```diff
@@ -0,0 +1,43 @@
+MyApp.get "/" do
+	@cat_1, @cat_2 = Cat.all.sample(2)
+  erb :"home"
+end
+
+MyApp.post "/cat_like/:winner/:loser" do
+	@winner = Cat.find(params[:winner])
+	@loser = Cat.find(params[:loser])
+	@winner.likes += 1
+	@winner.save
+  redirect :"like/#{@winner.id}/#{@loser.id}"
+end
+
+MyApp.get "/like/:winner/:loser" do
+	@winner = Cat.find(params[:winner])
+	@loser = Cat.find(params[:loser])
+   erb :"like_added"
+end
+
+MyApp.get "/results" do
+	cat_list = Cat.all
+	@rank = cat_list.sort {|a,b| b.likes <=> a.likes}
+  erb :"results"
+end
+
+MyApp.get "/admin" do
+	@cat_list = Cat.ranking
```

Comment:
that was a typo. it should say @cat_list = Cat.all
I was working on a method before called 'ranking' and I think that was a remnant of me trying something